contrib_bots/lib/lib/John/var/database.db

```diff
@@ -0,0 +1 @@
+{"I would like to request a remote dev instance": {"created_at": {"$date": 1484202844835}, "extra_data": {}, "in_response_to": []}, "Joke!": {"created_at": {"$date": 1484202844838}, "extra_data": {}, "in_response_to": []}, "Contributors are more than welcomed! Please read\n    https://github.com/zulip/zulip#how-to-get-involved-with-contributing-to-zulip\n    to learn how to contribute.": {"created_at": {"$date": 1484202844825}, "extra_data": {}, "in_response_to": [{"text": "I want to contribute", "occurrence": 1}]}, "I am John, my job is to assist you with Zulip.": {"created_at": {"$date": 1484202844842}, "extra_data": {}, "in_response_to": [{"text": "What is your name?", "occurrence": 1}]}, "What is your name?": {"created_at": {"$date": 1484202844841}, "extra_data": {}, "in_response_to": []}, "I want to contribute": {"created_at": {"$date": 1484202844824}, "extra_data": {}, "in_response_to": []}, "I can provide useful information and jokes if you follow etiquette.": {"created_at": {"$date": 1484202844845}, "extra_data": {}, "in_response_to": [{"text": "What can you do?", "occurrence": 1}]}, "Greetings! You should receive a response from one of our mentors soon.\n    In the meantime, why don't you learn more about running Zulip on a development\n    environment? https://zulip.readthedocs.io/en/latest/using-dev-environment.html": {"created_at": {"$date": 1484202844837}, "extra_data": {}, "in_response_to": [{"text": "I would like to request a remote dev instance", "occurrence": 1}]}, "Only if you ask nicely!": {"created_at": {"$date": 1484202844839}, "extra_data": {}, "in_response_to": [{"text": "Joke!", "occurrence": 1}]}, "What can you do?": {"created_at": {"$date": 1484202844844}, "extra_data": {}, "in_response_to": []}, "What is Zulip?": {"created_at": {"$date": 1484202844827}, "extra_data": {}, "in_response_to": []}, "Zulip is a powerful, open source group chat application. Written in Python\n    and using the Django framework, Zulip supports both private messaging and group\n    chats via conversation streams. You can learn more about the product and its\n    features at https://www.zulip.org.": {"created_at": {"$date": 1484202844834}, "extra_data": {}, "in_response_to": [{"text": "What is Zulip?", "occurrence": 1}]}}
```

Comment:
I think this file should not be checked into GitHub. Every instance of the bot should have it's own version of the file.